File: Pratica 6 - Funcoes 2/cpp_logic/exercicio6_v2.cpp
#include <iostream>
using namespace std;

float desconto(float valor, float desc) {
    return valor - (valor * desc / 100);
}

int main() {
    float valor, desc;
    cout << "Digite o valor do produto: ";
    cin >> valor;
    cout << "Digite o desconto (%): ";
    cin >> desc;

    cout << "O valor final é: R$ " << desconto(valor, desc);

    return 0;
}

File: Pratica 6 - Funcoes 2/java_logic/Exercicio6_v2.java
import java.util.Scanner;

public class Exercicio6_v2 {
    static double aplicar_desc(double valor, double desc) {
        return valor - (valor * desc / 100);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite o valor do produto: ");
        double v = sc.nextDouble();
        System.out.println("Digite o valor do desconto(%): ");
        double d = sc.nextDouble();

        System.out.println("O valor final do produto é: " + aplicar_desc(v, d));

        sc.close();
    }
}
